luigi/parameter.py

```diff
@@ -1540,6 +1540,66 @@ def normalize(self, var):
                 var=var, choices=self._choices))
 
 
+class ChoiceListParameter(Parameter):
```

Comment:
This is fine with me with one exception. If this class is intended to be a multi-select of `ChoiceParameter`, it's counter to object-oriented for it to not inherit from `ChoiceParameter` (or a common Choice-based parent) such for them to maintain alignment of their choice parsing and validation.

What are your thoughts?